micro-infra-spring-base/src/main/groovy/com/ofg/infrastructure/metrics/config/IsGraphitePublishingEnabled.groovy

```diff
@@ -0,0 +1,22 @@
+package com.ofg.infrastructure.metrics.config
+
+import org.springframework.context.annotation.Condition
+import org.springframework.context.annotation.ConditionContext
+import org.springframework.core.type.AnnotatedTypeMetadata
+
+class IsGraphitePublishingEnabled implements Condition {
+
+    @Override
+    boolean matches(ConditionContext context, AnnotatedTypeMetadata metadata) {
+        return graphitePublishingSwitchNotDefined(context) || graphitePublishingIsEnabled(context)
+    }
+
+    private boolean graphitePublishingSwitchNotDefined(ConditionContext context) {
+        !context.environment.containsProperty('graphite-publishing')
```

Comment:
Cool. There are already some properties related to metrics (https://github.com/4finance/boot-microservice/blob/master/src/main/resources/application.yaml#L8-L10), may I suggest e.g. `graphite.publishing.enabled`?